### DISCUSSION TITLE
Can I just start vscode and expose the webdriver server on a port?

### DISCUSSION THREAD

**Discussion by bcpeinhardt** [redhat-developer/vscode-extension-tester#1533] [Q&A]
I'm curious if I can use ex-tester to build my plugin and then drive VSCode with a simple selenium client? I'm using a testing DSL that interprets as calls to a selenium client, and if I could simply point it at the right port and drive vscode that would be fabulous.

**Comment by djelinek** [comment]
Hello, I am afraid that it is not possible. There is no implementation for this usecase at the moment. 

See https://github.com/redhat-developer/vscode-extension-tester/blob/main/packages/extester/src/browser.ts#L113 which would needs to be modified to accept some external selenium client by webdriver (if it is possible on side of selenium-webdriver, which I don=t know)
